### PR TITLE
CPDRP-474: Private Beta

### PR DESCRIPTION
**Pull request by ethanmills**
### Context
We want to invite schools to join a private beta for adding participant details

### Changes proposed in this pull request
- Rake task to invite schools
- Send welcome email to induction tutor
- Turn on the feature flag for the school

### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

File: app/services/utm_service.rb [hunk 1]
```diff
@@ -9,11 +9,17 @@ class UTMService
     sign_in: "sign-in",
     challenge_partnership: "challenge-partnership",
     partnership_notification: "partnership-notification",
+    june_private_beta: "june-private-beta",
   }.freeze
 
-  def self.email(campaign)
+  SOURCES = {
+    service: "cpdservice",
+    private_beta: "cpdprivatebeta",
```
Comment (kicferk1) [inline app/services/utm_service.rb:17]: Do we expect a bunch of other emails / links and want to link them together under the private beta umbrella?

If not, is that `cpdprivatebeta` a bit of an overkill?

File: app/services/invite_schools.rb [hunk 2]
```diff
@@ -61,6 +61,42 @@ def send_ministerial_letters
...
+    )
+
+    school_urns.each do |urn|
+      school = School.find_by(urn: urn)
+      if school.nil?
+        logger.info "School not found, urn: #{urn} ... skipping"
+        next
+      end
+
+      if FeatureFlag.active?(:induction_tutor_manage_participants, for: school)
```
Comment (kicferk1) [inline app/services/invite_schools.rb:77]: And how do we turn it on, and for what schools?

File: app/services/utm_service.rb (hunk 1 above)
Comment (ethanmills) [reply app/services/utm_service.rb:17]: It was requested 🤷

File: app/services/invite_schools.rb (hunk 2 above)
Comment (ethanmills) [reply app/services/invite_schools.rb:77]: Line 88

File: app/services/utm_service.rb (hunk 1 above)
Comment (ethanmills) [reply app/services/utm_service.rb:17]: https://ukgovernmentdfe.slack.com/archives/C01J1J4FM50/p1623761868488000

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**